source/changelog.txt

```diff
@@ -12,6 +12,21 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.1.5
+------
+
+*Released February 19, 2024*
+
+- Fixed the continuous integration task release_publish(_dry_run) 
+  parameters.
+- Removed the e2e test from the unit test suite (:issue:`MONGOSH-1716`).
+- Modified ``shell.exec`` to avoid leaking any secrets 
+  (:issue:`MONGOSH-1704`).
+- Remove ``userId`` from identify and track (:issue:`COMPASS-7607`).
```

Comment:
None of these are user-facing changes, you'll probably just want to use a reference to the Node.js upgrade and reference the security announcements ([link](https://nodejs.org/en/blog/vulnerability/february-2024-security-releases/)) for that. You can look at the 2.0.2 release notes for comparison to see how we've handled that in the past :)